docroot/modules/custom/foia_webform/src/Plugin/QueueWorker/FoiaSubmissionQueueWorker.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Drupal\foia_webform\Plugin\QueueWorker;
+
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+use Drupal\webform\WebformSubmissionInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides base functionality for the FoiaForm Queue Workers.
+ */
+abstract class FoiaSubmissionQueueWorker extends QueueWorkerBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * The form submission.
+   *
+   * @var \Drupal\webform\WebformSubmissionInterface
+   */
+  protected $formSubmission;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(WebformSubmissionInterface $submission) {
```

Comment:
Same comment here - think this should be `WebformSubmissionStorageInterface $webformSubmissionStorage`